ts/features/bonus/cgn/__mock__/availableMerchants.ts

```diff
@@ -0,0 +1,32 @@
+export const availableMerchants = [
```

Comment:
```suggestion
export const availableMerchants : ReadonlyArray<TmpMerchantType> = [
```
So when you'll replace the tmp type, you'll exactly know where you have to update